README.md

```diff
@@ -1,9 +1,9 @@
 InMobi SDK for Android
 ======================
 
-Modified: November 03, 2016
+Modified: February 07, 2017
 
-SDK Version: 6.0.3
+SDK Version: 6.1.0
```

Comment:
The section **New in this version** also needs an update to include a summary of what's changed.